Examples/LJ_langevin.py

```diff
@@ -19,18 +21,29 @@
     lj_fluid.topology, sigma=sigma, epsilon=epsilon, cutoff=cutoff
 )
 
+
 from chiron.states import SamplerState, ThermodynamicState
 
+PRNG.set_seed(1234)
 # define the sampler state
 sampler_state = SamplerState(
-    x0=lj_fluid.positions, box_vectors=lj_fluid.system.getDefaultPeriodicBoxVectors()
+    x0=lj_fluid.positions,
+    current_PRNG_key=PRNG.get_random_key(),
+    box_vectors=lj_fluid.system.getDefaultPeriodicBoxVectors(),
+)
+
+velocities = initialize_velocities(
```

Comment:
In general, I think people will not have to directly interact with this, as the langevin integrator will initialize if the velocities haven't been set or if the number of atoms changed (or we just tell it to reinitialize each time we call the langevin move).   I think can just remove this from the example and rely on the langevin integrator to set it. 

I'll note, I put this as a separate function so (1) we don't duplicate code if we have another integrator that will utilize it and (2) to make it easier to directly test the velocity generation (so we don't end up with something like this mess: https://pubs.aip.org/aip/jcp/article/148/13/137101/76026/Comment-on-The-putative-liquid-liquid-transition ). 

Maybe it would make more sense to be the integrators.py file?